Prog/get-zillow.R

```diff
@@ -0,0 +1,125 @@
+         county %in% c('District of Columbia','Charles County','Frederick County','Montgomery County','Prince Georges County','Arlington County', 'Fairfax County', 'Fairfax City', 'Loudoun County','Prince William County', 'Alexandria City', 'Falls Church City', 'Manassas City', 'Manassas Park City')
+         ) %>%
+  arrange(county, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4))
+
+inventoryMSAJuly <- inventoryMSA %>% 
+  select(-RegionType, -SizeRank, -RegionID, -StateName) %>% 
+  rename(Metro = RegionName) %>% 
+  gather(key = 'monthx', value = 'inventoryMetro', -Metro) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  arrange(Metro, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-monthx, -Metro, -month)
+
+#### inventory for COG region
+inventoryCOGJuly <- inventoryCOGmonth %>% 
+    filter(month=="07") %>% 
+    group_by(year) %>% 
+    summarize(JulyInventory= sum(inventory))
+
+
+# Sale prices --------------------------------------------------------
+
+price <- read_csv('http://files.zillowstatic.com/research/public/Metro/Sale_Prices_Msa.csv')
+
+priceMSAJuly <- price %>% 
+  select(-RegionID, -SizeRank) %>% 
+  rename(name = RegionName) %>% 
+  gather(key = 'monthx', value = 'Mediansaleprice', -name) %>% 
+  filter(name =="Washington, DC") %>%
+  mutate(month = substr(monthx, 6, 7), 
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-name, -monthx, -month)%>% 
+  arrange(year, Mediansaleprice, -month)
+
+# Rent for SF and MF residents --------------------------------------------------------
+
+rentSF <- read_csv('http://files.zillowstatic.com/research/public/Metro/Metro_MedianRentalPrice_Sfr.csv')
+rentlargeMF <- read_csv('http://files.zillowstatic.com/research/public/Metro/Metro_MedianRentalPrice_Mfr5Plus.csv')
+rentcondo <-read_csv('http://files.zillowstatic.com/research/public/Metro/Metro_MedianRentalPrice_CondoCoop.csv')
+rentduplex <-read_csv('http://files.zillowstatic.com/research/public/Metro/Metro_MedianRentalPrice_DuplexTriplex.csv')
+
+
+MetroRentSF <- rentSF %>% 
+  rename(Metro = RegionName) %>% 
+  select(-SizeRank) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  gather(key = 'monthx', value = 'MedianSFRent', -Metro) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  arrange(Metro, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-monthx, -Metro, -month)
+
+
+MetroRentMF <- rentlargeMF %>% 
+  rename(Metro = RegionName) %>% 
+  select(-SizeRank) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  gather(key = 'monthx', value = 'MedianMFRent', -Metro) %>% 
+  filter(Metro=="Washington, DC") %>% 
+  arrange(Metro, monthx) %>% 
+  mutate(month = substr(monthx, 6, 7),
+         year= substr(monthx, 1,4)) %>% 
+  filter(month=="07") %>% 
+  select(-monthx, -Metro, -month)
```

Comment:
i might leave in the month variable, just to be more transparent about how you annualized this (fine to take one month, just good to know what month it is. just don't merge on the month and it should run fine.

Reply:
We only need July.